ERCS/erc-7835.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7835
+title: Universal On-Chain Metadata Parser Interface
+description: A universal interface for parsing wallet-accessed metadata, enhancing security, usability, and data readability.
+author: ZeroKPunk (@ZeroKPunk), 0xbbPizza (@0xbbPizza)
+discussions-to: https://ethereum-magicians.org/t/erc-7835-standard-metadata-abstract-parsing-interface-for-evm-wallet/22175
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-12-03
+---
+
+## Abstract
+
+This proposal defines a standardized interface for parsing on-chain metadata (MetaData) relevant to Wallets. The standard consists of two core components:
+
+- **1.Wallet-level parser interface:** wallet_metaDataAbstractParser
```

Comment:
Inline code should be enclosed in backticks (\`) throughout the document. Also, just use a regular markdown ordered list.

```suggestion
1. Wallet-level parser interface:** `wallet_metaDataAbstractParser`
```